README.md

```diff
@@ -152,6 +155,13 @@ hab_service "core/redis" do
   strategy 'rolling'
   topology 'standalone'
 end
+
+# Create a runtime user for a service.
```

Comment:
This example looks a bit weird. Maybe add a comment that explains it a bit better as I'm not sure whether this is related to the "if not creating hab user, here is what you'd do to create a user" Also there are double quotes all over in the examples :)